### ISSUE TITLE
I'm moving to NY for a bit! Looking for more collaborators as I may have to put this on pause for awhile

### ISSUE THREAD

**Issue by sortofsleepy** [sortofsleepy/arkit-tests#31]
Hey all! 

So some news happened for me over the holidays - looks like I'm moving back to NYC for a bit at the end of the month!  I'll admit, I do have a bit of trepidation given the recent bomb cyclone ❄️ , but excited nonetheless 😄.

My family still has Macs, but I recently switched to Windows this year for my personal computer -  I've been able to keep working on this project every once in awhile thanks to the fact that the (soon to be ex) company that currently employees me gives everyone Macs and a device to test things on.

With me moving to NY - I obviously loose access to both those things 😢 which means I may not be able to update anything till at least May. 

I'm hopeful that the place I'm gonna be working for will allow me to borrow a Mac from time to time but I'm not counting on it so I'm putting this notice up ahead of time in case anyone would like to help out with maintaining this repo.

Obviously I can't just add anyone to the collab list but as long as you seem like a pretty decent programmer and appear to have a understanding of OpenGL/WebGL/openFrameworks, I'll be happy to add you to the list if you're interested in helping.

Let me apologize ahead of time for the poor support you may receive if issues come up. If I think I can make changes without breaking anything I'll do so, but for the most part this project will likely get put on hold until I have access to a Mac and iPhone again.

In case you'd like to help but don't know where to start, one thing that could use work since I haven't gotten around to finding the time to put more effort into would be more examples. 

I will of course still be more than happy to answer questions about the design of the addon(which could probably use some improvement on as well). 

Thank you for your understanding. If you're in NYC and would like to hang out, please feel free to say hi too!

**Comment by sortofsleepy** [comment]
@aferriss - seems there's no way to attach a message to it but I sent you a collab invite. You don't have to help maintain this if you don't want to of course, but since you seem to pretty actively use this; figured it'd be easier to just give you push access in case you wanted to make changes.

**Comment by aferriss** [comment]
Thanks for all your work on this Joe, good luck on your travels and move!

**Comment by ofZach** [comment]
I'm in nyc, lets to meet up!  also happy to help with examples.   good luck with the move!

**Comment by sortofsleepy** [comment]
@ofZach for sure! Would love to hang out and see what you're up to some time while I'm there. And thanks for being willing to help!

**Comment by sortofsleepy** [comment]
So I got my hands on a Mac! That being said - it's kinda missing the important part of the power adapter on top of the previous owner still needing some things saved on it, thus not leaving much room to do to much(there's only like 10 gigs on the SSD :/ ) but I imagine I can still keep up with this from time to time(commute and work doesn't leave much free time sadly enough). 

Closing due to having a mac now! But still, if you're interested in collaborating, you're more than welcome to ping me.
